power_grid_model_c/power_grid_model/include/power_grid_model/math_solver/common_solver_functions.hpp

```diff
@@ -57,22 +59,60 @@ template <symmetry_tag sym> inline void copy_y_bus(YBus<sym> const& y_bus, Compl
     });
 }
 
+template <symmetry_tag sym>
+inline void truncate_to_zero(double const& theta, ComplexValue<sym> const& val1, ComplexValue<sym> const& val2,
+                             ComplexValue<sym>& result) {
```

Comment:
please provide a docstring. while it's not difficult to understand, the rationale behind the need for this function may not be straightforward. Someone may encounter this and be like "why would I need this? wouldn't the calculation be more accurate without it?" and remove it. Or, worse, they may not even want to touch this code in the first place.

Reply:
The code may be straightforward, but I totally agree that the reason why it is being done in such a way is not. Well noted, and will add it.